scripts/build/TestPlatform.Dependencies.props

```diff
@@ -13,8 +13,8 @@
     from a build parameter would not be available, so I am writing this version from the build.ps1 script to keep it in sync -->
     <NETTestSdkVersion>17.2.0-dev</NETTestSdkVersion>
 
-    <MSTestFrameworkVersion>2.1.0</MSTestFrameworkVersion>
-    <MSTestAdapterVersion>2.1.0</MSTestAdapterVersion>
+    <MSTestFrameworkVersion>2.2.8</MSTestFrameworkVersion>
+    <MSTestAdapterVersion>$(MSTestFrameworkVersion)</MSTestAdapterVersion>
```

Comment:
This is so that we have both versions in sync because I can't really think of a case where we would want different versions.

Reply:
This can't be done, integration tests are reading the file.